src/window.js

```diff
@@ -56,6 +56,4 @@ export default (win) => {
   if (!win.createImageBitmap) win.createImageBitmap = createImageBitmap;
 
   mockPrototype();
-
-  return win;
```

Comment:
keep this line.

Reply:
There is no reason to keep this. You didn't use return value, right?

Reply:
1. the function get the `window`, and return a new window. But this method is special, it is mutable, return the same `window`
2. break change for strict mode

Reply:
I can't fully agree with you but updated as you wish.

This is not my main issue to discuss, I just want to fix this bug for Node.js 18 ASAP.